zulipterminal/ui_tools/views.py

```diff
@@ -982,12 +993,16 @@ def keypress(self, size: urwid_Size, key: str) -> str:
 class StreamInfoView(PopUpView):
     def __init__(self, controller: Any, color: str,
                  desc: str, title: str) -> None:
+        stream_info_content = [('', [desc])]
+        _, column_widths = self.calculate_table_widths(
+            stream_info_content, len(title))
         # Add 4 (for 2 Unicode characters on either side) to the popup title
         # length to make sure that the title gets displayed even when the
         # content is shorter than the title length (+4 Unicode characters).
-        width = max(len(desc) + 2, len(title) + 4)
-        stream_info_content = [urwid.Text(desc, align='center')]
-        super().__init__(controller, stream_info_content, 'STREAM_DESC', width,
+        popup_width = max(len(desc) + 2, len(title) + 4)
```

Comment:
@kaustubh-nair Note that the +2 is extra width which we had was for the center alignment. We might want to remove it as we're not using the alignment with `Text`.

Reply:
(See also https://github.com/zulip/zulip-terminal/pull/695#discussion_r456789654.)